### PR TITLE
Add basic possibility to vote on videos.

### PR DESCRIPTION
**Pull request by jann**
This is still very basic and ajax is not  supported yet. But maybe its a good point to merge to master and start new branches for the remaining stuff. 
What do you think?

### CODE REVIEW

File: web/controllers/vote_controller.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
+defmodule SignDict.VoteController do
+  use SignDict.Web, :controller
+
+  plug Guardian.Plug.EnsureAuthenticated, handler: SignDict.GuardianErrorHandler
+
+  alias SignDict.Vote
+  alias SignDict.Video
+
+  def create(conn, %{"video_id" => video_id}) do
+    IO.puts "creating"
```
Comment (EinLama) [inline web/controllers/vote_controller.ex]: This line should be removed. Unless you have any special reason for this being here? 😄 Looks like an older debugging statement.

**Comment by EinLama** [comment]
I like the idea of merging this before enhancing the feature further. This can be done in the next PR 😸

File: web/controllers/vote_controller.ex (hunk 1 above)
Comment (jann) [inline web/controllers/vote_controller.ex]: @EinLama thanks for reviewing. You were right, this was debug output. 
(This)[https://github.com/signdict/website/pull/56/commits/5b92235f7e933419dfaaff19b45647620d9f6d9d] commit solves it 🎉

**Comment by jann** [comment]
@EinLama @bitboxer This is solved. Feel free to review again and merge. I continue working on that during next days in a new branch...

**Comment by bitboxer** [comment]
Thanks for this. It is merged now!

**Pull request merged**